base/applications/sndvol32/sndvol32.c

```diff
@@ -1275,6 +1276,21 @@ MainWindowProc(HWND hwnd,
             break;
         }
 
+        case WM_HOTKEY:
+        {
+            if (wParam == 1) 
```

Comment:
Please add a define in the header file `sndvol32.h` for value `1` here and in other places where referenced.